### DISCUSSION TITLE
ComposedChart tooltip style

### DISCUSSION THREAD

**Discussion by severinlandolt** [recharts/recharts#4774] [Q&A]
Hi! I am working on composed charts, e.g. https://recharts.org/en-US/examples/LineBarAreaComposedChart

The tooltip uses the hover style of the line/area charts. Is it possible to use the tooltip hover from the BarChart instead? Thanks!
![CleanShot 2024-07-15 at 18 44 01@2x](https://github.com/user-attachments/assets/8215f8d5-eed2-46c2-b8af-6d0a207a74b6)

**Comment by ckifer** [comment]
Hi!

the `cursor` prop on `Tooltip` is supposed to be for supplying your own cursor shape. 

But https://codesandbox.io/p/sandbox/simple-composed-chart-forked-3wgmjs?file=%2Fsrc%2FApp.tsx%3A82%2C9
I can't really get it to work unfortunately unless I just give it a `rect` which doesn't take positioning into account.

probably a bug

**Comment by JoaquinPalacios** [comment]
Hi, I managed to get it working by passing the strokeWidth as a prop. This way, instead of being a thin line, the cursor now displays at the desired width. Here's my implementation:

````
cursor={{
  width: 32,
  stroke: "oklch(0.129 0.042 264.695)",
  strokeWidth: 32,
  strokeOpacity: 0.1,
}}
````

This cursor object needs to be passed as a prop to the Tooltip.

Hope this helps!
